parlai/mturk/core/mturk_manager.py

```diff
@@ -1310,8 +1310,7 @@ def soft_block_worker(self, worker_id):
         """Soft block a worker by giving the worker the block qualification"""
         qual_name = self.opt['block_qualification']
         assert qual_name != '', ('No block qualification has been specified')
-        self.give_worker_qualification(worker_id, qual_name,
-                                       is_sandbox=self.is_sandbox)
+        self.give_worker_qualification(worker_id, qual_name)
```

Comment:
Why did this change?

Reply:
``is_sandbox`` is not a keyword arg for ``self.give_worker_qualification`` (though it is an argument for the ``mturk_client``; that's handled in ``self.give_worker_qualification``).

Reply:
Oh oops I don't think I've ever directly used manager.give_worker_qualification, hence the bug. Thanks for the fix!